website/docs/docs/cloud/git/connect-github.md

```diff
@@ -25,19 +25,21 @@ Connecting your GitHub account to dbt Cloud provides convenience and another lay
 You can connect your dbt Cloud account to GitHub by installing the dbt Cloud application in your GitHub organization and providing access to the appropriate repositories. 
 To connect your dbt Cloud account to your GitHub account: 
 
-1. Navigate to **Your Profile** settings by clicking the gear icon in the top right. 
+1. From dbt Cloud, click on your account name in the left side menu and select **Account settings**. 
 
-2. Select **Linked Accounts** from the left menu.
+2. Select **Personal profile** under the **Your profile** section.
 
-<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-connect.gif" title="Navigated to Linked Accounts under your profile"/>
+3. Scroll down to **Linked Accounts**.
 
-3. In the **Linked Accounts** section, set up your GitHub account connection to dbt Cloud by clicking **Link** to the right of GitHub. This redirects you to your account on GitHub where you will be asked to install and configure the dbt Cloud application. 
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-connect.png" title="Navigated to Linked Accounts under your profile"/>
```

Comment:
non blocker but the screenshot you added is of when the user has already linked their github account. i wonder if we can instead have a screenshot that hasn't been linked so it matches that actual steps you're writing out here.

Reply:
Thanks @mirnawong1 I'll see if I can find one